src/convertToObject.js

```diff
@@ -12,7 +13,16 @@
  * @return {object}
  */
 function convertToObject(sourceString) {
-  // write your code here
+  const result = {};
+  const normalized = sourceString.split(':').map(x => x.trim()).join(': ');
+
+  normalized.split(';').map(x => x.trim()).filter(y => y !== '').forEach(z => {
+    const [key, value] = z.split(': ');
```

Comment:
checklist p.4  - All chained methods called on the same level should have same indentation. This makes code much more readable.

Reply:
Also, variable names such as `x, y, z` are bad

Reply:
fixed